.github/workflows/wheel_lgpu_arm.yml

```diff
@@ -0,0 +1,124 @@
+
+concurrency:
+  group: wheel_macos_arm64-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  mac-set-matrix-arm:
+    if: |
+      github.event_name != 'pull_request' ||
+      contains(github.event.pull_request.labels.*.name, 'ci:build_wheels') ||
+      github.event_name == 'workflow_dispatch'
+    timeout-minutes: 45
+    name: Set builder matrix
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout PennyLane-Lightning
+        uses: actions/checkout@v4
+
+      - name: Python version
+        id: pyver
+        run: |
+          if [[ ${{ github.event_name }} == 'pull_request' ]]; then
+            echo "python_version=$(python3 scripts/gen_pyver_matrix.py \
+              --min-version=3.${{ env.PYTHON3_MIN_VERSION }} \
+              --max-version=3.${{ env.PYTHON3_MAX_VERSION }})" >> $GITHUB_OUTPUT
+          else
+            echo "python_version=$(python3 scripts/gen_pyver_matrix.py \
+              --min-version=3.${{ env.PYTHON3_MIN_VERSION }} \
+              --max-version=3.${{ env.PYTHON3_MAX_VERSION }} --range)" >> $GITHUB_OUTPUT
+          fi
+
+    outputs:
+      python_version: ${{ steps.pyver.outputs.python_version }}
+
+  mac-wheels-arm64:
+    needs: [mac-set-matrix-arm]
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-12]
+        arch: [arm64]
+        pl_backend: ["lightning_gpu"]
+        cibw_build: ${{fromJson(needs.mac-set-matrix-arm.outputs.python_version)}}
+    timeout-minutes: 30
+    name: macos-latest::arm64 - ${{ matrix.pl_backend }} (Python ${{ fromJson('{ "cp39-*":"3.9","cp310-*":"3.10","cp311-*":"3.11","cp312-*":"3.12" }')[matrix.cibw_build] }})
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout PennyLane-Lightning
+        uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        name: Install Python
+        with:
+          python-version: '3.9'
+
+      - name: Install cibuildwheel
+        run: python -m pip install cibuildwheel~=2.16.0
+
+      - name: Build wheels
+        env:
+          CIBW_BUILD: ${{ matrix.cibw_build }}
+
+          # MacOS specific build settings
+          CIBW_BEFORE_ALL_MACOS: |
+            brew uninstall --force oclint
+
+          # Python build settings
+          CIBW_BEFORE_BUILD: |
+            python -m pip install pybind11 ninja cmake~=3.24.0 setuptools scipy
+
+          CIBW_ENVIRONMENT: |
+            CMAKE_ARGS="-DCMAKE_CXX_COMPILER_TARGET=arm64-apple-macos11 -DCMAKE_SYSTEM_NAME=Darwin -DCMAKE_SYSTEM_PROCESSOR=ARM64 -DENABLE_OPENMP=OFF" \
+            PL_BACKEND="${{ matrix.pl_backend }}"
+
+          CIBW_BEFORE_TEST: |
+            python -m pip install -r requirements-tests.txt
+            SKIP_COMPILATION=True PL_BACKEND="lightning_qubit" python -m pip install . -vv
+
+          CIBW_TEST_COMMAND: |
+            DEVICENAME=`echo ${{ matrix.pl_backend }} | sed "s/_/./g"`
+            pl-device-test --device=${DEVICENAME} --skip-ops -x --tb=short --no-flaky-report
```

Comment:
Do we have a test for this hardware configuration? 
How we are sure that the code will use the GPU of Mx chips?

Reply:
@LuisAlfredoNu Not yet as it requires more tests locally.. I'll tag you and others when it's ready to review :)